packages/cfd/src/Containers/mt5-compare-table-content.tsx

```diff
@@ -225,6 +233,8 @@ const DMT5CompareModalContent = ({
         poa_pending,
     } = getAuthenticationStatusInfo(account_status);
 
+    const is_other_countries = ['af', 'pk', 'mg'].includes(residence);
```

Comment:
is_other_countries? which other countries? 😄 
we should back this based on a logic or a requirement rather than a residency which can be added or removed any time.

Reply:
This whole thing will be removed as we are going to push the Revamp Compare Account